tests/utils/queryApiGateway.test.tsx

```diff
@@ -0,0 +1,150 @@
+import queryApiGateway, { buildApiUrl, extractJSON, Claim, QueryParams } from '../../utils/queryApiGateway'
+import mockEnv from 'mocked-env'
+import fs from 'fs'
+import fetch from 'node-fetch'
+
+// Mock some modules
+jest.mock('fs')
+jest.mock('node-fetch')
+// const { Response } = jest.requireActual('node-fetch')
```

Comment:
This line should be uncommented, but needs to pass linting before it can be commited